### PR TITLE
1.24-stable cherry-picks

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by tbrisker** [comment]
looks like we also need the nulldb adapter now

**Comment by mmoll** [comment]
[test foreman]

**Comment by mmoll** [comment]
[test foreman]

**Comment by mmoll** [comment]
[test foreman]

**Comment by mmoll** [comment]
[test foreman]

**Comment by mmoll** [comment]
[test foreman]

**Comment by mmoll** [comment]
finally :green_heart:

File: bundler.d/nulldb.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+gem 'activerecord-nulldb-adapter'
```
Comment (tbrisker) [inline bundler.d/nulldb.rb:1]: This means we need to cp the packaging to rpm/1.24

Comment (mmoll) [reply bundler.d/nulldb.rb:1]: For DEB I'm planning to just remove that file in the build process.

**Review by tbrisker — APPROVED** [review]
Thanks @mmoll !

**Pull request merged**